check.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# Print and run a command
+print_and_run() {
+    cmd=\`$@\`
+    echo Running $cmd...
+    $*
+    status_code=$?
+    if [[ $status_code -eq 0 ]] ; then
+        echo $cmd was succesful!
+    else
+        echo $cmd failed!
+    fi
+    return $status_code
+}
+
+#
```

Comment:
Should this be a comment? ("If no arguments", etc)

Reply:
Indeed